VolleyPaint/Assets/Scripts/Ability/SpikeBall.cs

```diff
@@ -0,0 +1,25 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[CreateAssetMenu(menuName ="Litkey/Ability/SpikeBall")]
+public class SpikeBall : Ability
+{
+    Transform ball;
+
+    // stops the ball for a second and spikes the ball to the ground diagnally
+    public override void UseAbility(GameObject parent)
+    {
+        base.UseAbility(parent);
+        ball = GameObject.FindGameObjectWithTag("Ball").transform;
+        //parent.GetComponent<PlayerShoot>()
+        
+    }
+
+    public override void BeginCooldown(GameObject parent)
+    {
+        base.BeginCooldown(parent);
+    }
+
+    
+}
```

Comment:
Also this could just be making your shot more powerful/have more force

or just remove this and have it be part of the super jump script

Reply:
Agree. But we can make new script instead. Not really using this script.